### DISCUSSION TITLE
Hosting gratuito--Recomendacion?

### DISCUSSION THREAD

**Discussion by PDamianReyes** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#376] [Q&A]
Hola, comunidad, esperaba que me pudieran aconsejar con un Hosting gratuito para poder subir las pagians de pasteles y vacuanacion (Se que no es necesario pero si me gustaria), eh encontrado varios pero ninguno me convence y me preguntaba si ustedes estaban usando alguno en especifico???

**Comment by hackit92** [comment]
Yo te recomendaría probar con AWS, Oracle o Firebase, sería lo mejor para que puedas conocer mejor el manejo de servidores web

**Reply by PDamianReyes** [reply]
Gracias, lo checare ahora mismo!!

**Comment by doguedogue** [comment]
GitHub pages
